modules/scenario/mmd.json

```diff
@@ -0,0 +1,303 @@
+{
```

Comment:
Please remove the scenario module export. The module definition now comes from the Simulator solution pack